pages/faqs/risks.en.md

```diff
@@ -11,7 +11,8 @@ With this in mind, it is easy to see an ERC-20 stablecoin with an [audited contr
 
 ### Empty Set Dollar Risks
 
-Participating in Empty Set Dollar has its risks. Mainly, it is important to remember that ESD is an **early stage monetary experiment** whose value is not derived from any collateral, but instead careful incentives, game theory, and most importantly, others’ trust. There is a possibility that ESD’s system fails, which will realize value and IL risk to the fullest (i.e. price goes to above 1 USDC).
+Participating in Empty Set Dollar has its risks. Mainly, it is important to remember that ESD is an **early stage monetary experiment** whose value is not derived from any collateral, but instead careful incentives, game theory, and most importantly, others’ trust. There is a possibility that ESD’s system fails, which will realize value and IL risk to the fullest (i.e. ESD becomes worthless).
+* Note that ESD eventually plans to become partially collateralized (see [here](https://medium.com/@scott_lew_is/a-vision-for-empty-set-dollar-ce29b7e0e297)).
```

Comment:
Given the decentralised nature of the protocol, a medium post by a community member doesn't constitute the community's plans to change the protocol. 

Please soften the language to suggest that there are people in the community who have sought a grant to explore alternative mechanism designs.

Reply:
Done.